src/Administration/Resources/app/administration/src/app/assets/scss/global.scss

```diff
@@ -144,11 +134,28 @@ a[target="_blank"]:not(.sw-external-link, .sw-internal-link, .mt-button, .mt-lin
     margin-bottom: 24px;
 }
 
+// TODO: remove later
```

Comment:
what should that mean? todo tags should always contain a issue number. otherwise nobody will remember to fix those ever